cmd/tdex/main.go

```diff
@@ -30,6 +30,19 @@ var (
 	statePath   = path.Join(tdexDataDir, "state.json")
 )
 
+func init() {
+	dataDir := os.Getenv("TDEX_OPERATOR_DATADIR")
+	if len(dataDir) <= 0 {
+		return
+	}
+
+	if !path.IsAbs(dataDir) {
```

Comment:
mmm why? lets resolve if is relative

Reply:
I replicated the same behavior of btcd and lnd CLI's.. the path is expanded if starting with `~`. The relative path is allowed but not expanded.